pkgs/racket-doc/scribblings/style/textual.scrbl

```diff
@@ -144,10 +144,11 @@ read code on monitors that accommodate close to 250 columns, and on
 occasion, our monitors are even wider. It is time to allow for somewhat
 more width in exchange for meaningful identifiers.
 
-So, when you create a file, add a line with @litchar{;; } followed by ctrl-U 99 and
-@litchar{-}. When you separate "sections" of code in a file, insert the same line.
-These lines help both writers and readers to orient themselves in a file.
-In scribble use @litchar|{@; }| as the prefix.
+So, when you create a file, add a line with @litchar{;; } followed by ctrl-U 99
+and @litchar{-}. @margin-note{In Vi, the command is 99a- followed by Esc.} When
```

Comment:
How does this render? I recall there's a difference between `margin-note` and `margin-note*`. Is this the right one to use?

Reply:
Ah, I think you're right that this should be `margin-note*` given its use mid-paragraph. Will fix and check rendering.

Reply:
You were correct; fixed. Range-diff (not hidden because it's small):
```
1:  907d3ea366 = 1:  907d3ea366 docs: capitalize the noun Git
2:  1d248982d7 ! 2:  665e037505 docs: mention the Vi command to add sections
    @@ pkgs/racket-doc/scribblings/style/textual.scrbl: read code on monitors that acco
     -These lines help both writers and readers to orient themselves in a file.
     -In scribble use @litchar|{@; }| as the prefix.
     +So, when you create a file, add a line with @litchar{;; } followed by ctrl-U 99
    -+and @litchar{-}. @margin-note{In Vi, the command is 99a- followed by Esc.} When
    ++and @litchar{-}. @margin-note*{In Vi, the command is 99a- followed by Esc.} When
     +you separate "sections" of code in a file, insert the same line. These lines
     +help both writers and readers to orient themselves in a file. In scribble use
     +@litchar|{@; }| as the prefix.
3:  adce70f37d = 3:  808676897e docs: link fx+ and unsafe-fx+
4:  7823976663 = 4:  ca7d2a2a56 docs: correct Git pull command
5:  7521dcb753 = 5:  1108c95343 docs: unquote "merge commit"
6:  af33479301 = 6:  1374b3b095 docs: italicize "e.g."
7:  b082353e9d = 7:  8f3f1cd517 docs: correct macro body
```